core/Db/Schema/Mysql.php

```diff
@@ -299,6 +299,21 @@ public function getTablesCreateSql()
                                       ) ENGINE=$engine DEFAULT CHARSET=utf8
             ",
 
+            'archive_invalidations' => "CREATE TABLE `{$prefixTables}archive_invalidations` (
+                                            idinvalidation BIGINT UNSIGNED NOT NULL AUTO_INCREMENT,
+                                            idarchive INTEGER UNSIGNED NULL,
+                                            name VARCHAR(255) NOT NULL,
+                                            idsite INTEGER UNSIGNED NOT NULL,
+                                            date1 DATE NOT NULL,
+                                            date2 DATE NOT NULL,
+                                            period TINYINT UNSIGNED NOT NULL,
+                                            ts_invalidated DATETIME NULL,
+                                            status TINYINT(1) UNSIGNED DEFAULT 0,
+                                            PRIMARY KEY(idinvalidation),
+                                            INDEX index_idsite_dates_period_name(idsite, date1, date2, period, name)
```

Comment:
btw be the index maybe better `idsite,period,date1` eg as used in the `getNextInvalidatedArchive ()` query?

Reply:
that might work, will have to check it's ok for ranges in case I query for individual invalidations... I don't think I do...